src/core/entities/MRHyperlinkEntity.js

```diff
@@ -53,7 +53,7 @@ export default class MRHyperlinkEntity extends MRTextEntity {
      * @description Callback function of MREntity - makes sure the link object is created and sets up event
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?